privacy-security-explainer.md

```diff
@@ -57,9 +60,182 @@ Rounding, quantization, and fuzzing are three categories of mitigations that mod
 Limiting is when data is reported only when it is within a specific range. For example, it is possible to comfortably limit reporting positional pose data when a user has moved beyond a specific distance away from an approved location. Care should be taken to ensure that the user experience is not negatively affected when employing this mitigation. It is often desireable to avoid a 'hard stop' at the at the end of a range as this may cause disruptive user experiences.
 
 # Protected functionality
-**TODO** Fill this in based on the information in [#638](https://github.com/immersive-web/webxr/pull/638) so that it adequately addresses [#393](https://github.com/immersive-web/webxr/issues/393) and [#485](https://github.com/immersive-web/webxr/issues/485).
+The sensitive information exposed via WebXR can be divided into categories that share threat profiles and necessary protections against those threats.
+
+## Immersiveness
+Users must be in control of when immersive sessions are created because the creation causes invasive changes on a user's machine. For example, starting an immersive session will engage the XR device sensors, take over access to the device's display, and begin presentation which may terminate another application's access to the XR hardware. It may also incur significant power or performance overhead on some systems or trigger the launching of a status tray or storefront.
+
+Developers indicate the desire to create an immersive session by passing `immersive-vr` or `immersive-ar` into `xr.requestSession()`. 
+
+```js
+// VR button click handler
+function onVRClick() {
+  xr.requestSession('immersive-vr').then(onVRSessionCreated);
+}
+
+//AR button click handler
+function onARClick() {
+  xr.requestSession('immersive-ar').then(onARSessionCreated);
+}
+```
 
-# Appendix A: Proposed IDL
-```webidl
-// TODO: Add any necessary IDL
-```
\ No newline at end of file
+In response, the UA must ensure that:
+* The function was invoked in response to a [user activation](#user-activation)
```

Comment:
> in response to a user activation

Nit: I *think* that this is probably correct grammer, but it sounds weird to me. :) Would "user activation event" work better?